scrnaseq.Rmd

```diff
@@ -169,20 +171,31 @@ param$downsample_cells_n = NULL
 
 # Path to git repository
 param$path_to_git = "."
+
+# Debugging mode: 
+# 'default_debugging' for default, 'terminal_debugger' for debugging without X11, 'print_traceback' for non-interactive sessions 
+param$debugging_mode = "default_debugging"
```

Comment:
We need to add it to the README.

Reply:
Can you add it? Or how do I include your changes to the README?

Reply:
I'll add it.